scripts/migrate_etcd_0.1_0.2.sh

```diff
@@ -19,5 +19,5 @@ fi
 ENV="$NS/$CLUSTER"
 echo "Copying namespace and placement data from env=default_env to env=$ENV"
 
-kubectl exec -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl get _sd.placement/default_env/m3db | kubectl exec -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl put "_sd.placement/$ENV/m3db"
-kubectl exec -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl get _kv/default_env/m3db.node.namespaces | kubectl exec -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl put "_kv/$ENV/m3db.node.namespaces"
+kubectl exec -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl get --print-value-only _sd.placement/default_env/m3db | head -c-1 | kubectl exec -i -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl put "_sd.placement/$ENV/m3db"
+kubectl exec -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl get --print-value-only _kv/default_env/m3db.node.namespaces | head -c-1 | kubectl exec -i -n "$ETCD_NS" "$ETCD_POD" -- env ETCDCTL_API=3 etcdctl put "_kv/$ENV/m3db.node.namespaces"
```

Comment:
I just realized this won't work on OSX:
```
$ uname
Darwin
$ echo foo | head -c-1
head: illegal byte count -- -1
```

I think we could get the length of the message with `wc -c`, subtract 1 from that as `NUM`, and then `head -c $NUM` and that would be cross-platform.

```
# test
$ kubectl exec -n m3db etcd-1 -- env ETCDCTL_API=3 etcdctl get --print-value-only _sd.placement/default_env/m3db | wc -c
    1987
$ kubectl exec -n m3db etcd-1 -- env ETCDCTL_API=3 etcdctl get --print-value-only _sd.placement/default_env/m3db | head -c 1986 | md5
577c64a5bb96ec18c857152d8336f02b
```

```
# control 
$ kubectl exec -n m3db etcd-1 -- env ETCDCTL_API=3 etcdctl get -w json _sd.placement/default_env/m3db | jq -r '.kvs[0].value' | base64 -D | md5
577c64a5bb96ec18c857152d8336f02b
```

What do you think? I'm happy to tack that onto this PR!